apps/api/src/app/notification-template/notification-template.controller.ts

```diff
@@ -43,7 +43,7 @@ import { CreateNotificationTemplateQuery } from './queries';
 @Controller('/notification-templates')
 @UseInterceptors(ClassSerializerInterceptor)
 @UseGuards(JwtAuthGuard)
-@ApiTags('Workflows')
+@ApiTags('Notification Templates')
```

Comment:
Changed this to `Notification templates` to show it as deprecated on the swagger
<img width="1511" alt="image" src="https://github.com/novuhq/novu/assets/29251776/8eeb7ec3-ffd2-43ca-bf4c-a4baaa52ab5f">

Reply:
It is in the different PR:https://github.com/novuhq/novu/pull/3701